Air pollution prediction/code.py

```diff
@@ -0,0 +1,50 @@
+import requests
+city = 'kanpur'
+url = 'http://api.waqi.info/feed/' + city + '/?token='
+api_key = '37f96394ffe8b6cca1110af3d8270604c711c688'
```

Comment:
You have exposed your api key here , either replace it like this:-
```suggestion
api_key = '<Enter your api key here>' 
```
Or create a .env file and place the api key there